packages/react-core/src/components/Progress/ProgressHelperText.tsx

```diff
@@ -0,0 +1,34 @@
+import * as React from 'react';
+import progressStyle from '@patternfly/react-styles/css/components/Progress/progress';
+import { HelperText, HelperTextItem, HelperTextItemProps } from '../HelperText';
+import { ProgressProps } from './Progress';
+
+export interface ProgressHelperTextProps extends React.HTMLProps<HTMLDivElement> {
+  /** Content rendered inside the progress helper text component. */
+  children?: React.ReactNode;
+  /** Variant of the progress component. */
+  progressVariant?: ProgressProps['variant'];
+}
+
+export const ProgressHelperText: React.FunctionComponent<ProgressHelperTextProps> = ({
+  children,
+  progressVariant
+}: ProgressHelperTextProps) => {
+  const helperTextVariantMapping: {
+    [Property in ProgressProps['variant']]: HelperTextItemProps['variant'];
+  } = {
+    success: 'success',
+    warning: 'warning',
+    danger: 'error'
+  };
+
+  return (
+    <div className={progressStyle.progressHelperText}>
+      <HelperText isLiveRegion>
```

Comment:
Yeah I could see situations where it may not be needed so I'm not against adding a prop for that.

Do you have a preference for defaulting it to be applied or not? I feel like more use cases would benefit from it being a live region than not so I _think_ I lean towards having the default behavior be to apply it.

Reply:
I think the only reason I might suggest it _not_ being applied by default is to follow how AlertGroup and HelperText both default to their live regions not being applied by default (ignoring cases where an `aria-live` is hardcoded rather than controlled via prop, like ClipboardCopy's internal Tooltip).

Reply:
In the most recent changes I removed the default `isLiveRegion`. If a consumer wishes to enable that behavior they can do so by passing a helper text component with that prop set to true with my current implementation that allows any ReactNode to be passed and rendered.